src/fmeval/eval_algorithms/helper_models/helper_model.py

```diff
@@ -123,7 +140,16 @@ def get_helper_scores(self, text_input: List[str]) -> Dict[str, List[float]]:  #
         :param text_input: list of text inputs for the model
         :returns: dict with keys as score name and value being list of scores for text inputs
         """
-        return self._model(text_input)
+        inputs = self._tokenizer(text_input, return_tensors="pt", truncation=True, padding=True).to(self._model.device)
+        scores = torch.sigmoid(self._model(**inputs)[0]).cpu().detach().numpy()
```

Comment:
It's unclear if this is supposed to work in a batch call or not. why do you select `self._model(**inputs)[0]`? Are we assuming `text_input` is a list with only one string?

Reply:
`self._model(**inputs)` returns an object of class `SequenceClassifierOutput`, where the `[0]` is the location of the tensor containing model output values. The tensor can be two dimensional so batching is still supported here.

This was referenced from detoxify repo's [`predict` method](https://github.com/unitaryai/detoxify/blob/master/detoxify/detoxify.py#L117-L123).

Reply:
Ok thanks! Please add some inline comments for future reference :)